### PR TITLE
Repository setup eric closes #5

### LINKED ISSUE #5: Repository Set Up - Eric
DOD (Updated for Sprint 2): A repository interface is set up for the admin approval service.  
DOD: A repository to store submissions and is populated with dummy submission data

### PR DESCRIPTION
**Pull request by RoEric**
Temporary, placeholder models were used. Code needs to incorporate Willie's entity models after his branch is merged to the develop branch.

### CODE REVIEW

**Review by sruble — COMMENTED** [review]
looks good to me Eric.

File: photo-points/Models/SeedData.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Builder;
+
+namespace photo_points.Models
```
Comment (weberwilliamb) [inline photo-points/Models/SeedData.cs]: I noticed that you hadn't updated with our entity model we merged last week.

**Review by weberwilliamb — CHANGES_REQUESTED** [review]

**Review by MaxIgnatyev — APPROVED** [review]
Looks good to me.

File: photo-points/Models/ApplicationDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace photo_points.Models
+{
+    public class ApplicationDbContext : DbContext
+{
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
+            : base(options) { }
+        public DbSet<Submission> Submissions { get; set; }
+}
+}
```
Comment (Sara-webaplication) [inline photo-points/Models/ApplicationDbContext.cs]: looks ok

**Review by weberwilliamb — APPROVED** [review]
I'm not seeing any issues.

**Review by MaxIgnatyev — APPROVED** [review]
Looks good

**Review by sruble — APPROVED** [review]
Looks good Eric

**Review by Sara-webaplication — APPROVED** [review]

**Comment by Sara-webaplication** [comment]
I approved it.

**Pull request merged**